tests/utils/test_format.py

```diff
@@ -142,6 +142,11 @@ def test_to_snake_case(input: str, output: str):
         ("kebab-case", "kebabCase"),
         ("kebab-case-two", "kebabCaseTwo"),
         ("snake_kebab-case", "snakeKebabCase"),
+        ("_hover", "_hover"),
+        ("-starts-with-hyphen", "-startsWithHyphen"),
```

Comment:
Is there a use case where `-starts-with-hyphen` is a valid variable name? 
Won't it clash with the `-` minus symbol?

Reply:
Yeah I don't think it can ever start with that

Reply:
Will merge this in as is since we should never hit this case really